core/server/middleware/serve-shared-file.js

```diff
@@ -8,12 +8,13 @@ var crypto = require('crypto'),
 // Handles requests to robots.txt and favicon.ico (and caches them)
 function serveSharedFile(file, type, maxAge) {
```

Comment:
I was thinking about that, but not sure.. Would `servePublicFile` be good? Oh... renaming...